### PR TITLE
FEATURE - Add ubuntu aws setup steps for broker

### PR DESCRIPTION
**Pull request by dannypaz**
## Description
This PR includes setup and install scripts for a hosted broker on AWS Ubuntu 16.04 LTS EC2

## Related PRs
List related PRs if applicable

## Todos
- [n/a] Tests
- [x] Documentation
- [x] Link to Trello

### CODE REVIEW

**Comment by dannypaz** [comment]
[AWS EC2 setup scripts for broker](https://trello.com/c/6qYu4qWI/667-aws-ec2-setup-scripts-for-broker)

File: setup-scripts/ubuntu-ec2-install.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
+########################################################
+# Sparkswap hosted broker installation for an AWS Ubuntu 16.04 LTS EC2 instance
+#
+# After `ubuntu-ec2-setup.sh` has been ran on your current machine, we are now able
```
[the remark this answers is not in the document]
Comment (dannypaz) [reply setup-scripts/ubuntu-ec2-install.sh]: @martineehrlich midwest accent is rich here

File: setup-scripts/ubuntu-ec2-setup.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+sudo apt-get update
+sudo apt-get install -y docker-ce python-setuptools python-pip build-essential libssl-dev
+
+# Installing NVM for use w/ the CLI
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.11/install.sh | bash
+
+# Install our node version which is 8.11.x
+nvm install 8.11 --latest-npm
+
+# We use pip to install the latest docker-compose instead of apt-get
```
[the remark this answers is not in the document]
Comment (dannypaz) [reply setup-scripts/ubuntu-ec2-setup.sh]: @martineehrlich ive updated the file for reasoning. It took more steps to install/upgrade and apt-get is currently out of date

File: .gitignore [hunk 3]
```diff
@@ -67,4 +67,4 @@ typings/
 proto/*
 
 # Override file which we use dynamically in relayer setup
-./docker-compose.override.yml
+docker-compose.override.yml
```
Comment (martineehrlich) [inline .gitignore]: I think if you merge master this change is already made.

**Pull request merged**